lib/services/files-api-service.js

```diff
@@ -0,0 +1,129 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+var fs = require('fs');
+var path = require('path');
+var walking = require('walk');
+
+module.exports = filesApiServiceFactory;
+
+di.annotate(filesApiServiceFactory, new di.Provide('Http.Services.Api.StaticFiles'));
```

Comment:
The source file name doesn't well match with this injectable name, suggest you to change the file name to something like `static-files-api-service`